dissect/target/plugins/os/unix/_os.py

```diff
@@ -159,6 +159,14 @@ def users(self, sessions: bool = False) -> Iterator[UnixUserRecord]:
                     _target=self.target,
                 )
 
+    @internal
+    def misc_user_paths(self) -> Iterator[tuple[str, tuple[str, str] | None]]:
+        if (root_path := self.target.fs.path("root")).exists():
+            yield (root_path, ("uid", 0))
+
+        if (home_path := self.target.fs.path("home")).exists():
```

Comment:
```suggestion
        if (root_path := self.target.fs.path("/root")).exists():
            yield (root_path, ("uid", 0))

        if (home_path := self.target.fs.path("/home")).exists():
```
It doesn't make any functional difference but visually it helps discern quicker what the intent is.